pkg/gatewayserver/gatewayserver.go

```diff
@@ -171,6 +173,36 @@ func New(c *component.Component, conf *Config, opts ...Option) (gs *GatewayServe
 	}
 
 	hooks.RegisterUnaryHook("/ttn.lorawan.v3.NsGs", rpclog.NamespaceHook, rpclog.UnaryNamespaceHook("gatewayserver"))
+	if conf.BasicStation.FallbackFrequencyPlanID != "" {
+		ctx = frequencyplans.WithFallbackID(ctx, conf.BasicStation.FallbackFrequencyPlanID)
```

Comment:
Can we scope this `ctx` to the BS frontend so that it doesn't influence frontends that we may add in the future?